types/lucene/index.d.ts

```diff
@@ -10,20 +10,37 @@ export interface TermLocation {
     offset: number;
 }
 
-export interface Node {
-    boost: null | number;
+export interface NodeField {
```

Comment:
Is there any point in exporting `NodeField`? I think it's more of an implementation detail than anything else - `NodeTerm` and `NodeRangedTerm` are using it themselves, but I don't see a need in exporting this `interface` to consumers directly.

Reply:
Not really.  I didn't use it in my code.  I only exported on the weak principle of allowing users to build polymorphic handlers, but they can do that via unions directly.  I'll strip the export.

Reply:
Either I'm not seeing the latest changes or something else, because it looks like the `NodeField` and `NodeTerm` are still `export`-ed from the diff I see.

I do see the `type`s being changes to `interface`s though.